rootfs/scheduler/__init__.py

```diff
@@ -1371,6 +1371,19 @@ def _pod_liveness_status(self, pod):
 
         return True
 
+    def _pod_ready(self, pod):
+        """Combines various checks to see if the pod is considered up or not by checking probes"""
+        if (
```

Comment:
Yeah, I had some other plans (due to a different PR I'm working on) but I like your approach better. I'll update and make my local thing work with this